yarn-project/circuit-types/src/keys/key_store.ts

```diff
@@ -112,12 +96,25 @@ export interface KeyStore {
   getMasterIncomingViewingSecretKeyForPublicKey(masterIncomingViewingPublicKey: PublicKey): Promise<GrumpkinPrivateKey>;
 
   /**
-   * Retrieves public keys hash of the account
-   * @throws If the provided account address is not associated with any of the registered accounts.
-   * @param account - The account address to get public keys hash for.
-   * @returns A Promise that resolves to the public keys hash.
+   * Gets the key validation request for a given master public key hash and contract address.
+   * @throws If the account corresponding to the master public key hash does not exist in the key store.
+   * @param pkMHash - The master public key hash.
+   * @param contractAddress - The contract address to silo the secret key in the the key validation request with.
+   * @returns The key validation request.
    */
-  getPublicKeysHash(account: AztecAddress): Promise<Fr>;
```

Comment:
`getPublicKeysHash` was no longer used anywhere so I nuked it. I introduced it before the complete address was refactored so it makes sense it could be nuked now.